### PR TITLE
Add note on creating virtualenv and Fedora install docs

### LINKED ISSUE #5438: Review and update developer docs
We should routinely review the development environment docs, here:

https://docs.securedrop.org/en/stable/development/setup_development.html

with an eye to: 
- are all instructions still accurate and up-to-date when setting up an environment from scratch;
- are the instructions clear and well-organized;
- do they work across different environments (Linux [different distros], Mac, Windows).

@joaedwar has offered to take a stab at this in the next SecureDrop sprint; exact scope TBD.

### PR DESCRIPTION
**Pull request by nebelsprachen**
## Status

Ready for review 

## Description of Changes
Changes proposed in this pull request: Adds docs on installing in Fedora and other Linux distributions. Also, adds a note on creating a `virtulalenv` in Python3.  

This employs changes discussed in the [PR 5459](https://github.com/freedomofpress/securedrop/pull/5459)

Fixes #5438 .

Changes proposed in this pull request:

## Testing

How should the reviewer test this PR?
Write out any special testing steps here.

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing production instances.
2. New installs.

### If you made changes to the server application code:

- [x] Linting (`make lint`) and tests (`make test`) pass in the development container

### If you made changes to `securedrop-admin`:

- [ ] Linting and tests (`make -C admin test`) pass in the admin development container

### If you made changes to the system configuration:

- [ ] [Configuration tests](https://docs.securedrop.org/en/latest/development/testing_configuration_tests.html) pass

### If you made non-trivial code changes:

- [ ] I have written a test plan and validated it for this PR

### If you made changes to documentation:

- [x] Doc linting (`make docs-lint`) passed locally

### If you added or updated a code dependency:

Choose one of the following:

- [ ] I have performed a diff review and pasted the contents to [the packaging wiki](https://github.com/freedomofpress/securedrop-debian-packaging/wiki)
- [ ] I would like someone else to do the diff review

### CODE REVIEW

File: docs/development/documentation_guidelines.rst [hunk 1]
```diff
@@ -150,10 +152,10 @@ called ``securedrop``. This ``securedrop`` directory also contains a
...
      .
      ├── securedrop
-     │   │
-     │  ...
-     │   ├── securedrop
-    ... ...
+     │   │
+     │  ...
+     │   ├── securedrop
+    ... ...
```
Comment (eloquence) [inline docs/development/documentation_guidelines.rst:158]: This looks like an automatic conversion from nonbreaking spaces to regular spaces. Looks content-neutral to me in rendered output.

**Comment by eloquence** [comment]
LGTM! :) See #5459 for full review/edit history.

File: docs/development/setup_development.rst [hunk 2]
```diff
@@ -35,11 +37,32 @@ be installed via the official documentation links:
 * `Docker CE for Ubuntu`_
 * `Docker CE for Debian`_
 
-Make sure to follow the `Post-installation steps for Linux`_, as well.
+Make sure to follow the `post-installation steps for Linux`_.
+
+
+Fedora and Other Linux Distributions
```
Comment (eloquence) [inline docs/development/setup_development.rst]: "Other Linux Distributions" is a bit broad, unless we want to call out specific Fedora derivatives may be best to just say "Fedora Linux" here.

File: docs/development/setup_development.rst [hunk 3]
```diff
@@ -301,4 +324,4 @@ To configure a multi-machine evironment in Qubes, follow the Quick Start instruc
 create a standalone VM named ``sd-dev``, then follow the Linux instructions above to install the
 required packages, *omitting* Virtualbox.
 
-Then, complete the steps described in :doc:`qubes_staging`.
+Then, complete the steps described in :doc:`qubes_staging`.
```
Comment (conorsch) [inline docs/development/setup_development.rst]: Couldn't figure out what changed here, but after checking out locally, it's just a no-trailing-newline change. :+1:

File: docs/includes/virtualenv.txt [hunk 4]
```diff
@@ -0,0 +1,5 @@
+.. note:: You can install, upgrade, and remove Python packages using ``pip``, the Python package installer. By default, ``pip`` will attempt to install packages system-wide. Use ``venv`` or ``virtualenv`` to avoid installing Python packages globally and to isolate the installation of dependencies of a specific application.
+          
+          Run the command ``python3 -m venv .venv`` to create and manage your virtual environment. To begin using the virtual environment, use the command ``source .venv/bin/activate`` to activate.   
+        
+          You can also use `virtualenvwrapper <https://virtualenvwrapper.readthedocs.io/en/latest/install.html>`_ which provides the ``mkvirtualenv`` shell function and is a set of extensions to the ``virtualenv`` tool. The extensions include wrappers for creating and deleting virtual environments and otherwise managing your development workflow.
```
Comment (conorsch) [inline docs/includes/virtualenv.txt]: Great use of an include! Was tempted to close out the long-standing https://github.com/freedomofpress/securedrop/issues/1319 but there are a few pointers in there that may still be worth following up on.

**Review by conorsch — APPROVED** [review]
Looks great. Thank you for taking the time, @joaedwar!

**Comment by conorsch** [comment]
Just waiting on CI, fine tom erge after confirmed passing!

**Pull request merged**